Spotkanie3_Zadania/CurrencyConversion/CurrencyConversion.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Czy domergowanie zmian zwiazanych z CurrencyConversion jest tu potrzebne?

Reply:
Ty co ja znowu namieszałem?

Reply:
Chyba był najprawdopodobniej branch z brancha.

Reply:
To usuń wszytsko związane z **CurrencyConversion **